pkg/asset/internal/templates.go

```diff
@@ -25,6 +25,21 @@ contexts:
     user: {{ .UserName }}
 `)
 
+	KubeSystemSARoleBindingTemplate = []byte(`apiVersion: rbac.authorization.k8s.io/v1alpha1
```

Comment:
They are added to the default static assets in https://github.com/kubernetes-incubator/bootkube/pull/261/files#diff-4475804b8fcae522e91ffb4651f8f967R30, so are generated by the `render` command (`NewDefaultAssets`) then created by the `start` command (`CreateAssets`).